src/util.ts

```diff
@@ -0,0 +1,8 @@
+import IdIterator from 'json-rpc-random-id';
+
+export const createPayload = (data: any) => {
+  data.id = IdIterator();
```

Comment:
Curious about why do we need a specific random number? Couldn't it be just a uuid? [json-rpc spec](https://www.jsonrpc.org/specification#request_object) states that it could be a string

Reply:
Yep it can be, I just took this from blockaid teams reference PR.